docs/management/connectors/action-types/crowdstrike.asciidoc

```diff
@@ -0,0 +1,40 @@
+[[crowdstrike-action-type]]
+== Crowdstrike connector
+++++
+<titleabbrev>Crowdstrike</titleabbrev>
+++++
+:frontmatter-description: Add a connector that can use the Crowdstrike API to send actions.
+:frontmatter-tags-products: [kibana] 
+:frontmatter-tags-content-type: [how-to] 
+:frontmatter-tags-user-goals: [configure]
+
+preview::[]
+
+The Crowdstrike connector communicates with Crowdstrike Management Console via REST API.
+
+[float]
+[[define-crowdstrike-ui]]
+=== Create connectors in {kib}
+
+You can create connectors in *{stack-manage-app} > {connectors-ui}*. For example:
+
+[role="screenshot"]
+image::management/connectors/images/crowdstrike-connector.png[Crowdstrike connector]
+// NOTE: This is an autogenerated screenshot. Do not edit it directly.
+
+[float]
+[[crowdstrike-connector-configuration]]
+==== Connector configuration
+
+Crowdstrike connectors have the following configuration properties:
+
+ClientId::  A Crowdstrike API client id.
+ClientSecret::  A Crowdstrike API client secret created by the user.
+URL:: The Crowdstrike tenant URL. If you are using the <<action-settings,`xpack.actions.allowedHosts`>> setting, make sure the hostname is added to the allowed hosts.
```

Comment:
In addition to changing the name to camel case (`CrowdStrike`), this reorders the settings to match the order and labels from the UI:

```suggestion
CrowdStrike connectors have the following configuration properties:

CrowdStrike API URL:: The CrowdStrike tenant URL. If you are using the <<action-settings,`xpack.actions.allowedHosts`>> setting, make sure the hostname is added to the allowed hosts.
CrowdStrike Client ID::  A CrowdStrike API client ID.
Client Secret::  A CrowdStrike API client secret created by the user.
```